catalog/java/io/material/catalog/button/res/layout/cat_buttons_toggle_group_fragment.xml

```diff
@@ -101,35 +101,35 @@
         android:id="@+id/icon_only_group"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content">
-      <Button
```

Comment:
Why explicitly hardcode these? These should autoinflate to MaterialButton via MaterialComponentsViewInflater?

Reply:
oh okk. Rreverted.